staff/lab/ocf-tv

```diff
@@ -43,15 +43,23 @@ def wait_for_port(host, port, timeout=5, ssh_proc=None):
 
 
 def volume(args):
-    call(('ssh', args.host,
-          'export PULSE_SERVER=localhost; pactl set-sink-volume {} {}%'.format(args.sink, args.amount)))
+    cmd = 'export PULSE_SERVER=localhost; '
+
+    if args.amount is not None:
+        cmd += 'pactl set-sink-volume 0 {}%; '.format(args.amount)
+
+    # thanks SO for awk tricks
+    cmd += "pactl list sinks | awk '/Sink #0/{nr[NR+9]}; NR in nr' | cut -d' ' -f 6;"
```

Comment:
It'd be nice if we could find the proper way to print out this single field using `awk`.